tests/test_cli/test_summarize_table.py

```diff
@@ -17,14 +18,10 @@
 class TestSummarizeTable(TestCase):
 
     def setUp(self):
-        self.file = tempfile.NamedTemporaryFile()
-        self.file.write(biom1)
-        self.file.seek(0)
-        self.biom1 = self.file.name
-
-
-    def tearDown(self):
-        self.file.close()
+        with tempfile.NamedTemporaryFile() as fh:
```

Comment:
Probably easier to fake a file at this point, since `summarize_table` accepts a `Table` object now.